db/migrate/20230522044036_add_join_code_to_hideouts.rb

```diff
@@ -0,0 +1,6 @@
+class AddJoinCodeToHideouts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :hideouts, :join_code, :string
```

Comment:
Should be a number no?

Reply:
It's a hex code so string is required. I used hex instead of number because it makes the join code shorter and easier to type in.